<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveConfigPage</class>
 <widget class="QWidget" name="CurveConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2">
    <widget class="QListView" name="curveList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Curve Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lblLegendLabel">
        <property name="text">
         <string>Legend Label</string>
        </property>
        <property name="buddy">
         <cstring>editLegendLabel</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="editLegendLabel"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblLineStyle">
        <property name="text">
         <string>Line Style</string>
        </property>
        <property name="buddy">
         <cstring>comboLineStyle</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="LineStyleCombo" name="comboLineStyle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="spinLineWidth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string>pt</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="ColorButton" name="btnLineColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblSymbolStyle">
        <property name="text">
         <string>Symbol Style</string>
        </property>
        <property name="buddy">
         <cstring>comboSymbolStyle</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="SymbolStyleCombo" name="comboSymbolStyle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="spinSymbolSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string>pt</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="ColorButton" name="btnSymbolStyleColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblErrorBarStyle">
        <property name="text">
         <string>Error Bar Style</string>
        </property>
        <property name="buddy">
         <cstring>comboErrorBarStyle</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="LineStyleCombo" name="comboErrorBarStyle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QSpinBox" name="spinErrorBarWidth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string>pt</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="ColorButton" name="btnErrorBarStyleColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="4">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="lblScaleX">
        <property name="text">
         <string>Scale X</string>
        </property>
        <property name="buddy">
         <cstring>comboErrorBarStyle</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="spinScaleX"/>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="lblScaleY">
        <property name="text">
         <string>Scale Y</string>
        </property>
        <property name="buddy">
         <cstring>comboErrorBarStyle</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QDoubleSpinBox" name="spinScaleY"/>
      </item>
      <item row="10" column="0" colspan="4">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QSpinBox" name="spinEvery"/>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="lblEvery">
        <property name="text">
         <string>Every</string>
        </property>
        <property name="buddy">
         <cstring>comboErrorBarStyle</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="btnApplyTemplate">
        <property name="text">
         <string>&amp;Apply</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QComboBox" name="comboTemplates"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblFileName">
        <property name="text">
         <string>File Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLabel" name="lblFileNameDisplay">
        <property name="text">
         <string/>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineStyleCombo</class>
   <extends>QComboBox</extends>
   <header>saxsview_linestylecombo.h</header>
  </customwidget>
  <customwidget>
   <class>SymbolStyleCombo</class>
   <extends>QComboBox</extends>
   <header>saxsview_symbolstylecombo.h</header>
  </customwidget>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>saxsview_colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
